ospf6d/ospf6_gr_helper.c

```diff
@@ -1155,7 +1155,10 @@ DEFPY(show_ipv6_ospf6_gr_helper,
 	bool detail = false;
 
 	ospf6 = ospf6_lookup_by_vrf_name(VRF_DEFAULT_NAME);
-	OSPF6_CMD_CHECK_RUNNING();
+	if (ospf6 == NULL) {
```

Comment:
just wondering - that CHECK_RUNNING() macro is used in lots of places: why not enhance it to NULL-check "om6" and so forth?

Reply:
This `OSPF6_CMD_CHECK_RUNNING` is actually a no-op since the VRF support was added to OSPF6. I was going to completely remove it in a separate PR. I don't see any point in keeping and enhancing it – in every place where it is used, we already go through all the instances and check whether the one requested by the user exists.